docs/ref/models/constraints.txt

```diff
@@ -145,3 +145,24 @@ enforced immediately after every command.
 
     Deferred unique constraints may lead to a `performance penalty
     <https://www.postgresql.org/docs/current/sql-createtable.html#id-1.9.3.85.9.4>`_.
+
+``include``
```

Comment:
Is include supported just on Postgres? Is that said anywhere, other than 3.2.txt?

Reply:
I added note:
```
``include`` is supported only on PostgreSQL.
```